model/fieldtypes/compat/Float.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
```

Comment:
It means that you won't get a fatal error if you refer to class int without first referring to class dbint. The original approach suffered from that weakness.